geode-core/src/main/java/org/apache/geode/management/internal/configuration/validators/CacheElementValidator.java

```diff
@@ -43,13 +43,18 @@ public void validate(CacheElementOperation operation, CacheElement config)
   }
 
   private void validateCreate(CacheElement config) {
-    if ("cluster".equalsIgnoreCase(config.getGroup())) {
-      throw new IllegalArgumentException(
-          "'cluster' is a reserved group name. Do not use it for member groups.");
+    String group = config.getGroup();
+    if (CacheElement.CLUSTER.equalsIgnoreCase(group)) {
+      throw new IllegalArgumentException("'"
+          + CacheElement.CLUSTER
+          + "' is a reserved group name. Do not use it for member groups.");
     }
-    if (config.getGroup() != null && config.getGroup().contains(",")) {
-      throw new IllegalArgumentException(
-          "Group name should not contain comma.");
+    if (group != null && group.contains(",")) {
+      throw new IllegalArgumentException("Group name should not contain comma.");
+    }
+    String id = config.getId();
+    if (id.contains("/")) {
```

Comment:
because this validation is not specific to RegionConfig.  Any CacheElement that contains a / in it's ID will break our URI pattern.  I can move this fix to another PR if you prefer, as it was incidental cleanup not directly related to generics.